openlibrary/plugins/openlibrary/js/image_saved.js

```diff
@@ -0,0 +1,9 @@
+import { updateImage } from './image_change';
```

Comment:
Given size of this file and the fact it already uses image_change let's merge it with the one above.

I don't think our webpack configuration is clever enough to import from an imported bundled module without duplicating code anyway.